src/components/Feature.js

```diff
@@ -5,18 +5,37 @@ import FlagContext from '../utils/FlagContext';
 import Config from '../config';
 
 export class Feature extends Component {
+  componentDidMount() {
+    if (this.props.flags.isLDReady && this.props.onReady) {
+      this.props.onReady();
+    }
+  }
+
+  componentDidUpdate(prevProps) {
+    if (
+      !prevProps.flags.isLDReady &&
+      this.props.flags.isLDReady &&
+      this.props.onReady
+    ) {
+      this.props.onReady();
+    }
+  }
+
   render() {
-    const { children, flagId, variation, flags } = this.props;
+    const { children, flagId, variation, flags, waitForLD } = this.props;
 
     let variationMatch = false;
     if (Array.isArray(variation)) {
       variationMatch = variation.includes(flags[flagId]);
     } else if (['string', 'boolean', 'null'].includes(typeof variation)) {
       variationMatch = flags[flagId] === variation;
     }
-    const renderNode =
+
+    let renderNode =
       variation !== undefined ? variationMatch && children : children;
 
+    renderNode = waitForLD ? flags.isLDReady && renderNode : renderNode;
```

Comment:
Why not just do a return `null` here? No need to return the `FlagContext` wrapping `false`

Reply:
Good call